system-tests/src/test/java/org/eclipse/edc/samples/transfer/streaming/Streaming01httpToHttpTest.java

```diff
@@ -90,7 +90,7 @@ void setUp() throws IOException {
     @Test
     void streamData() throws IOException {
         var source = Files.createTempDirectory("source");
-        PROVIDER.registerDataPlane(List.of("HttpStreaming"), List.of("HttpData"));
+        PROVIDER.registerDataPlane(List.of("HttpStreaming"), List.of("HttpData"), List.of("HttpData-PUSH"));
```

Comment:
should that not be 
```suggestion
        PROVIDER.registerDataPlane(List.of("HttpStreaming"), List.of("HttpData-PULL"), List.of("HttpData-PUSH"));
```

Reply:
no, because in this method the second parameter is about "destination types", that don't contain the flow type. Anyway this registration will go away soon :)